bucket/ripme.json

```
@@ -1,13 +1,13 @@
 {
     "version": "1.7.95",
     "description": "Downloads image albums from various websites in bulk.",
-    "homepage": "https://github.com/RipMeApp/ripme",
+    "homepage": "https://github.com/ripmeapp2/ripme",
     "license": "MIT",
     "suggest": {
         "JRE": "java/temurin8-jre"
```

```suggestion
        "JRE": "java/temurin17-jre"
```

Requires Java 17 as per [description](https://github.com/ripmeapp2/ripme#about).